src/core/infer_response.h

```diff
@@ -138,12 +138,7 @@ class InferenceResponse {
     const std::vector<int64_t>& Shape() const { return shape_; }
 
     // Reshape the output tensor if specified in output config.
```

Comment:
Should elaborate the comment saying the function should be called only when output_config has reshape field. And keep this check in the function implementation? :
` if (output_config->has_reshape()) {`

Reply:
I'll update the comment. This function is only used locally so I'd rather avoid the additional check.